src/eval.c

```diff
@@ -244,6 +267,28 @@ static int eval7(char_u **arg, typval_T *rettv, int evaluate, int want_string);
 
 static int get_string_tv(char_u **arg, typval_T *rettv, int evaluate);
 static int get_lit_string_tv(char_u **arg, typval_T *rettv, int evaluate);
+static int get_template_string_tv(char_u *(*enclose)(char_u*), char_u *(*disclose)(char_u*), char_u *(*fix_quote)(char_u), int (*is_closing_quote)(char_u*), char_u **arg, typval_T *rettv, int evaluate);
+static char_u* disclose_quote(char_u *quoted);
```

Comment:
Please adjust the place of `*`.

Reply:
Sorry, what do you mean?
`static char_u *disclose_quote(char_u *quoted);` ?

Reply:
yes

Reply:
Thanks!
I fixed!